glib/src/object.rs

```diff
@@ -2263,16 +2263,13 @@ impl<T: ObjectType> ObjectExt for T {
     }
 
     fn try_set_property<V: ToValue>(&self, property_name: &str, value: V) -> Result<(), BoolError> {
-        let pspec = match self.find_property(property_name) {
-            Some(pspec) => pspec,
-            None => {
-                return Err(bool_error!(
-                    "property '{}' of type '{}' not found",
-                    property_name,
-                    self.type_()
-                ));
-            }
-        };
+        let pspec = self.find_property(property_name).ok_or_else(|| {
```

Comment:
Can you put all this cleanup into a separate PR? That's going to be merged faster and keeps this PR smaller then :)